presto-docs/src/main/sphinx/functions/decimal.rst

```diff
@@ -56,6 +56,12 @@ Binary Arithmetic Decimal Operators
     the precision and scale of the result data type,
     then an exception condition is raised - ``Value is out of range``.
 
+    Semantic for performing operations on different decimal types like ``DECIMAL(38, 0)``
```

Comment:
Thanks for rephrase.

As for the `decimal(38, 1)` is a super type of `decimal(38, 0)`. We agree that this is broken. There are also other issues with types, e.g: non compliance with SQL standard and various problems with character types. We have a plan how we would like to tackle the type system holistically, so it's possible to express various semantics. I think you recall our discussion about character type and INSERT INTO. Discussion about types (and how to make it aligned better with SQL standard) would be one of key points of @losipiuk, @ilfrin trip to FB at the end February.